neutrino.go

```diff
@@ -252,10 +252,9 @@ func (sp *ServerPeer) pushSendHeadersMsg() error {
 }
 
 // OnVerAck is invoked when a peer receives a verack bitcoin message and is used
-// to send the "sendheaders" command to peers that are of a sufficienty new
-// protocol version.
+// to kick start communication with them.
 func (sp *ServerPeer) OnVerAck(_ *peer.Peer, msg *wire.MsgVerAck) {
-	sp.pushSendHeadersMsg()
```

Comment:
Doesn't seem like it was ever used, pushed a commit to remove it.

Reply:
How wasn't it ever used? It signals to the other peer that they can send us headers as soon as a block is connected without us requesting it up front.

Reply:
This reduces the number of round trips needed to sync to tip:
  * without send headers: recv inv, send getheaders, get header, fetch filter, recv filter (2.5 RTT)
  * with send header: receive header,  fetch filter, recv filter (1.5 RTT)

Reply:
> How wasn't it ever used?

Looking at `git blame` it was introduced in the initial commit, but it was commented out. I'm not sure why that was the case, but wouldn't enabling this cause additional bandwidth as well since every peer we're connected to would send headers to us? It'd be a minimal amount though.

Reply:
It's an extra 11 KB or so per peer a day, but that's assuming we're always online. I think it's worth it to tend the node to a synced state more regularly. At that point, I think we commented it out as we weren't sure yet it the logic we had in place at that time would be able to handle spontaneous header messages. As a result, we should confirm this before we move forward with re-enabling the feature.